src/renderer/qssWidget/js/qssOpenUsbWidget.js

```diff
@@ -0,0 +1,350 @@
+            menuControls: ".flc-qssOpenUSBWidget-controls",
+            footerTip: ".flc-qssOpenUSBWidget-footerTip"
+        },
+        enableRichText: true,
+        activationParams: {},
+        closeDelay: 1200,
+        components: {
+            repeater: {
+                // whenever the setting changes (e.g. if the change is made via the PSP)
+                type: "gpii.psp.repeater",
+                container: "{openUSB}.dom.menuControls",
+                options: {
+                    model: {
+                        disabled: "{openUSB}.model.disabled",
+                        value: "{openUSB}.model.setting.value",
+                        styles: "{openUSB}.model.setting.styles"
+                    },
+                    modelRelay: {
+                        items: {
+                            target: "items",
+                            singleTransform: {
+                                type: "fluid.transforms.free",
+                                func: "gpii.qssWidget.openUSB.getRepeaterItems",
+                                args: [
+                                    "{openUSB}.model.setting"
+                                ]
+                            }
+                        }
+                    },
+                    dynamicContainerMarkup: {
+                        container: "<div role=\"radio\" class=\"%containerClass fl-qssWidgetMenu-item fl-focusable\" tabindex=\"0\"></div>",
+                        containerClassPrefix: "flc-qssWidgetMenu-item"
+                    },
+                    handlerType: "gpii.qssWidget.openUSB.presenter",
+                    markup: null,
+                    styles: {
+                        disabled: "disabled"
+                    },
+                    events: {
+                        onItemFocus: null
+                    },
+                    invokers: {
+                        updateValue: {
+                            funcName: "gpii.qssWidget.openUSB.handleOpenUSB",
+                            args: [
+                                "{arguments}.0", // value,
+                                "{openUSB}",
+                                "{arguments}.1", // keyboardEvent
+                                "{channelNotifier}.events.onQssOpenUsbRequested", // Mount USB event
+                                "{channelNotifier}.events.onQssUnmountUsbRequested", // Unmount USB event
+                                "{openUSB}.model.messageChannel"
+                            ]
+                        }
+                    },
+                    listeners: {
+                        "onCreate.enable": {
+                            this: "{that}.container",
+                            method: "removeClass",
+                            args: ["{that}.options.styles.disabled"]
+                        }
+                    }
+                }
+            },
+            closeTimer: {
```

Comment:
This block also part of the cut and paste from qssMenuWidget. Refactor as with the focus material referenced below